server/basicServices/basicServices/BasicServicesService.js

```diff
@@ -99,9 +99,12 @@ exports.embedYourself = function (body, user, originator, xCorrelator, traceIndi
       let isOldApplicationTcpClientUpdated = false;
       let oldApplicationTcpClientUuid;
       let oldApplicationForwardingTag = "PromptForEmbeddingCausesRequestForBequeathingData";
+      let isOldReleaseExist = await isForwardingNameExist(oldApplicationForwardingTag);
+      let oldApplicationNameInConfiguration;
+      if(isOldReleaseExist){
```

Comment:
Format the code please. Missing spaces and wrong indentation.

Reply:
Formatted the entire file.